skopi/detector/simple_square.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, N_pixel, det_size, det_distance, beam=None):
         sep = float(det_size) / N_pixel
         x = -det_size/2 + sep/2 + ar*sep
         y = -det_size/2 + sep/2 + ar*sep
-        X, Y = xp.meshgrid(x, y, indexing='xy')
+        Y, X = xp.meshgrid(x, y, indexing='xy')
```

Comment:
x and y are exactly the same here
therefore X and Y must be the same
so how does changing X,Y to Y,X help?

Reply:
I looked into condor and they swapped X and Y.
```
def generate_xypix(self, cx=None, cy=None):
        Y, X = numpy.meshgrid(numpy.float64(numpy.arange(self._ny))-(0. if cy is None else cy),
                              numpy.float64(numpy.arange(self._nx))-(0. if cx is None else cx),
                              indexing="ij")
        return X, Y
```

Reply:
they are using indexing="ij" and we are using indexing="xy" which is equivalent to a transpose

>>> x=range(3)
>>> y=range(3)
>>> xv, yv = np.meshgrid(x, y, sparse=False, indexing='ij')
>>> xv
array([[0, 0, 0],
       [1, 1, 1],
       [2, 2, 2]])
>>> yv
array([[0, 1, 2],
       [0, 1, 2],
       [0, 1, 2]])
>>> xv, yv = np.meshgrid(x, y, sparse=False, indexing='xy')
>>> xv
array([[0, 1, 2],
       [0, 1, 2],
       [0, 1, 2]])
>>> yv
array([[0, 0, 0],
       [1, 1, 1],
       [2, 2, 2]])

Reply:
changed to
```
        X, Y = xp.meshgrid(x, y, indexing='ij')
        Xar, Yar = xp.meshgrid(ar, ar, indexing='ij')
```
![image](https://user-images.githubusercontent.com/23045561/112402662-2cedda80-8cca-11eb-8843-0375d2468b1d.png)